src/styles/blocks/card.scss

```
@@ -0,0 +1,69 @@
+.card {
+  width: $card-width;
+  max-height: 407px;
+  color: ($main-color);
+  border: 1px solid #e5e5e5;
+  border-radius: ($card-border-radius);
+  padding: 32px 16px 16px;
+
+  &__pic {
+    width: 160px;
+    height: 134px;
+    display: block;
+    margin: 0 auto 40px;
+  }
+
+  &__title {
+    @include main-text-style;
+    line-height: 18px;
+    margin: 0 0 4px;
+    padding: 0;
+  }
+
+  &__id {
+    @include card-id-feedback-fonts;
+    color: ($id-price-label-color);
+    margin: ($code-price-margins);
+  }
+
+  &__review {
+    @include flex-betweeen;
+
+    align-items: center;
+    font-weight: ($font-weight-regular);
+    margin-bottom: 24px;
+  }
+
+  &__feedbacks {
+    @include card-id-feedback-fonts;
+  }
+
+  &__price {
+    @include flex-betweeen;
+    margin: ($code-price-margins);
+  }
+
+  &__button {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    text-decoration: none;
+    padding: 12px 69px;
+    max-width: 166px;
+    max-height: 40px;
+    line-height: 16px;
+    font-size: 14px;
+    font-weight: ($font-weight-bold);
```

you don't have to wrap it
```suggestion
    font-weight: $font-weight-bold;
```